src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -3,7 +3,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+@EnableJpaRepositories(basePackages = "wolox.training.repositories")
```

Comment:
This annotation works like the `@EntityScan` annotation, you can read my comment [here](https://github.com/wolox-training/fdn-java/pull/5#discussion_r398509770)